### ISSUE TITLE
Refactor itemDepartment HAL Link for new Responsible Collections Logic

### ISSUE THREAD

**Issue by clarkepeterf** [project-lux/lux-middletier#38]
**Problem Description:**
Currently the logic for the itemDepartment HAL link returns a collection set for an object. 

**Expected Behavior/Solution:**
itemDepartment Hal link is updated to return the collection set for the object

**Requirements:**
This change will require changing the HAL link to use the facets endpoint instead of the search endpoint. This will require an additional ticket on the frontend, which was not discussed previously, as well as marklogic issue (linked below)

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

~~- [ ] Wireframe/Mockup - Mike~~
~~- [ ] Committee discussions - Sarah~~
~~- [ ] Feasibility/Team discussion - Sarah~~
~~- [ ] Backend requirements - TBD~~
- [x] Frontend requirements- [frontend ticket](https://github.com/project-lux/lux-frontend/issues/132) @kamerynB 
~~- [ ] Questions~~
~~- List of questions for discussions. Answers should be documented within the issue.~~

**UAT/LUX Examples:**

~~- Example endpoints from LUX to use for development and testing, if applicable.~~

**Dependencies/Blocks:**

**Related Github Issues:**
- https://github.com/project-lux/lux-frontend/issues/153
- https://github.com/project-lux/lux-marklogic/issues/68
- [ ] Frontend issue - https://github.com/project-lux/lux-frontend/issues/132

**Related links:**

N/A
**Wireframe/Mockup:**
N/A

**Comment by roamye** [comment]
@clarkepeterf @kamerynB what is itemDepartment returning? is it responsible unit or something else?

**Comment by roamye** [comment]
as soon as we get any of the other data sets this will be important. labeled as normal but should still be prioritized.

**Comment by kamerynB** [comment]
@roamye Based on the original comment it says it "returns any agent which curates a set containing an object." From the [Reichstag](https://lux-front-tst.collections.yale.edu/view/object/64a6f0c2-6d58-44d8-8ffe-43ff5d213b62) record, the `itemDepartment` HAL link returns [this query](https://lux-front-tst.collections.yale.edu/api/search/agent?q=%7B%22curated%22%3A%7B%22containing%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux.collections.yale.edu%2Fdata%2Fobject%2F64a6f0c2-6d58-44d8-8ffe-43ff5d213b62%22%7D%7D%7D). You can view it in the advanced search [here](https://lux-front-tst.collections.yale.edu/view/results/people?q=%7B%22curated%22%3A%7B%22containing%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux.collections.yale.edu%2Fdata%2Fobject%2F64a6f0c2-6d58-44d8-8ffe-43ff5d213b62%22%7D%7D%7D).

I hope this helps.

**Comment by clarkepeterf** [comment]
@kamerynB I updated this HAL link ,but I realized that the frontend doesn't use it, because it just uses the `useGetCollectionQuery` on Objet pages. So I think our options are either to modify the frontend to use the HAL link, or to delete the HAL link. I think modifying the frontend is more in line with the spirit of how the frontend is meant to work:

The frontend shouldn't have to know what queries to construct for an object. It should be fed queries from the middle tier. Are you good with that plan?
